src/Sylius/Component/Promotion/Processor/PromotionProcessor.php

```diff
@@ -25,18 +25,41 @@
  */
 class PromotionProcessor implements PromotionProcessorInterface
 {
-    protected $repository;
+    /**
+     * @var PreQualifiedPromotionsProviderInterface
+     */
+    protected $activePromotionsProvider;
```

Comment:
I think name should be ``$preQualifiedPromotionsProvider`` because "active" is a concrete implementation.